packages/core/src/dialects/postgres/query.js

```diff
@@ -9,14 +9,17 @@ const { logger } = require('../../utils/logger');
 const debug = logger.debugContext('sql:pg');
 
 export class PostgresQuery extends AbstractQuery {
-  async run(sql, parameters) {
+  async run(sql, parameters, options) {
     const { connection } = this;
+    const shouldMinifyAliases = options?.shouldMinifyAliases ?? this.sequelize.options.minifyAliases;
```

Comment:
With the change I'm proposing above, you won't need the global option here either, the local option will be what controls everything